### PR TITLE
ruff ruff

### PR DESCRIPTION
**Pull request by hwchase17**

### CODE REVIEW

**Comment by agola11** [comment]
🚀

File: pyproject.toml [hunk 1]
```diff
@@ -74,9 +74,8 @@ responses = "^0.22.0"
 pytest-asyncio = "^0.20.3"
 
 [tool.poetry.group.lint.dependencies]
-flake8-docstrings = "^1.6.0"
 isort = "^5.10.1"
```
Comment (charliermarsh) [inline pyproject.toml:77]: If you want, you can also remove `isort` here, remove the `[tool.isort]` section below, then add:

```toml
[tool.ruff]
select = [
  "E",  # pycodestyle
  "F",  # pyflakes
  "I",  # isort
]
```

File: Makefile [hunk 2]
```diff
@@ -27,7 +27,7 @@ lint:
 	poetry run mypy .
 	poetry run black . --check
 	poetry run isort . --check
```
Comment (charliermarsh) [inline Makefile:29]: If you enable `isort`, you can remove this.

Comment (charliermarsh) [reply Makefile:29]: Then you can change your `poetry run isort .` under `format` to:

```
poetry run ruff --select I --fix .
```

**Comment by charliermarsh** [comment]
💯 💯 💯

File: Makefile (hunk 2 above)
Comment (hwchase17) [reply Makefile:29]: nice

**Pull request merged**